_data/Damian96/Damian96.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Damian
+institution: City College, International Faculty of the University of Sheffield
+profile_pic: Damian96.png
+quote:  Red 💊 or blue 💊; know your enemy or live in peace;
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the special character / emojis from your quote to maintain the same format